types/stripe-js/radar.d.ts

```diff
@@ -0,0 +1,24 @@
+import * as api from '../api';
+
+/* A Radar Session is a snapshot of the browser metadata and device details that helps Radar make more accurate predictions on your payments. 
+This metadata includes information like IP address, browser, screen or device information, and other device characteristics. 
+You can find more details about how Radar uses this data by reading about how Radar performs [advanced fraud detection](https://stripe.com/docs/disputes/prevention/advanced-fraud-detection).
+*/
+export type RadarSessionPayload =
```

Comment:
This can actually be put in the `stripe.d.ts` file [here](https://github.com/stripe/stripe-js/blob/19cdc3ee89a788be9ea90fc5558fd2f59d952b77/types/stripe-js/stripe.d.ts#L1011)!